simulations/vip-306/bscmainnet.ts

```diff
@@ -0,0 +1,52 @@
+import { expect } from "chai";
+import { BigNumber, Contract } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { expectEvents } from "../../src/utils";
+import { forking, testVip } from "../../src/vip-framework";
+import vip306, { COMMUNITY_WALLET, TOKEN_REDEEMER, TREASURY, VUSDC, VUSDC_AMOUNT } from "../../vips/vip-306/bscmainnet";
+import ERC20_ABI from "./abi/ERC20.json";
+import VTreasurey_ABI from "./abi/VTreasury.json";
+
+const USDC = "0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d";
+
+forking(38884900, () => {
+  let usdc: Contract;
+  let vusdc: Contract;
+  let prevBalanceCommunityWallet: BigNumber;
+  let prevTreasuryBalance: BigNumber;
+
+  before(async () => {
+    usdc = new ethers.Contract(USDC, ERC20_ABI, ethers.provider);
+    vusdc = new ethers.Contract(VUSDC, ERC20_ABI, ethers.provider);
+    prevBalanceCommunityWallet = await usdc.balanceOf(COMMUNITY_WALLET);
+    prevTreasuryBalance = await vusdc.balanceOf(TREASURY);
+  });
+
+  testVip("VIP-306 Transfer bootstrap liquidity needed for Arbitrum one to the Community wallet", vip306(), {
+    callbackAfterExecution: async txResponse => {
+      await expectEvents(txResponse, [VTreasurey_ABI], ["WithdrawTreasuryBEP20"], [1]);
+    },
+  });
+
+  describe("Post-VIP behavior", async () => {
+    it("should decrease vtreasury balance by the amount used to transfer vtoken corresponding to 25000 usdc", async () => {
+      expect(await vusdc.balanceOf(TREASURY)).to.closeTo(prevTreasuryBalance.sub(VUSDC_AMOUNT), parseUnits("700", 8));
+    });
+
+    it("Should increase USDC balance of the community wallet", async () => {
+      const currUSDcBal = await usdc.balanceOf(COMMUNITY_WALLET);
+      const delta = currUSDcBal.sub(prevBalanceCommunityWallet);
+      expect(delta).to.be.closeTo(parseUnits("25000", 18), parseUnits("10", 18));
```

Comment:
Shouldn't it be exactly equal now that we use `redeemUnderlyingAndTransfer`?

Reply:
Yes, It will be fixed in the following commit:
https://github.com/VenusProtocol/vips/pull/286/commits/7f02b385d08f11ec34cb8c5a39813478b712ec5a